components/butterfly/src/rumor/heat.rs

```diff
@@ -537,7 +537,7 @@ mod tests {
                 let heat_map = inner
                     .get(&RumorKey::from(*s))
                     .expect("Should have had a service rumor present");
-                for m in &[member_1_id, member_2_id, member_3_id] {
+                for _m in &[member_1_id, member_2_id, member_3_id] {
```

Comment:
Why `_m` instead of just `_`?  It's not like `m` was super descriptive to begin with, so I don't think much clarity is lost 😁